calico-cloud_versioned_docs/version-3.16/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -56,18 +63,18 @@ To be included in scan results, images must:
 
    :::note
 
-   You must download and set the executable flag each time you get a new version of the CLI.
+   You must download and set the executable flag each time you get a new version of the scanner.
 
    :::
 
-3. Verify the CLI scanner works correctly by running the version command.
+3. Verify that the scanner works correctly by running the version command.
 
    ```shell
    ./tigera-scanner version
    {{imageassuranceversion}}
    ```
 
-### Scan images using CLI
+### Scan images using the CLI
```

Comment:
Is the goal to remove CLI entirely? If so, it's still mentioned here.

Reply:
Removed.